pkg/config/remote/uptane/client.go

```diff
@@ -205,7 +112,7 @@ func (c *Client) unsafeTargets() (data.TargetFiles, error) {
 	if err != nil {
 		return nil, err
 	}
-	return c.directorTUFClient.Targets()
+	return c.director.tufClient.Targets()
 }
```

Comment:
it would be nice to have a comment explaining why some functions have "unsafe" in their name (several other occurrences in the same file)

Reply:
Will add it in followup RCM-115